### ISSUE TITLE
Error with tidy lm.beta objects - Column 6 must be named

### ISSUE THREAD

**Issue by ldmahoney** [tidymodels/broom#810]
Hello! I know this issue was previously discussed in issue #496, but I am coming across it now. @mattle24 took a look at my reprex, error traceback, and session info and suggested I open a new issue. I appreciate anyone who is able to take a look and help me with this!

Here's what I ran
``` r
reprex::reprex({
  library(broom)
  library(lm.beta)
  
  mod <- stats::lm(speed ~ ., data = cars)
  std <- lm.beta(mod)
  tidy(std)
}, si = TRUE)

rlang::last_error()
```
And here's what I got. 
``` r
library(broom)
#> Warning: package 'broom' was built under R version 3.5.3
library(lm.beta)

mod <- stats::lm(speed ~ ., data = cars)
std <- lm.beta(mod)
tidy(std)

#> Error: Column 6 must be named.
#> Use .name_repair to specify repair.

rlang::last_error()

<error/rlang_error>
Column 6 must be named.
Use .name_repair to specify repair.
Backtrace:
  1. generics::tidy(std)
  2. broom:::tidy.lm(std)
  3. broom:::tidy.summary.lm(s)
  4. broom::fix_data_frame(co, nn[1:ncol(co)])
  6. tibble:::as_tibble.data.frame(ret)
  7. tibble:::as_tibble.list(unclass(x), ..., .rows = .rows, .name_repair = .name_repair)
  8. tibble:::lst_to_tibble(x, .rows, .name_repair, col_lengths(x))
  9. tibble:::set_repaired_names(x, .name_repair)
 14. tibble:::repaired_names(names(x), .name_repair = .name_repair)
 15. tibble:::check_unique(new_name)
```
<sup>Created on 2020-01-17 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

<details>

<summary>Session info</summary>

``` r
devtools::session_info()
#> - Session info ---------------------------------------------------------------
#>  setting  value                       
#>  version  R version 3.5.2 (2018-12-20)
#>  os       Windows 10 x64              
#>  system   x86_64, mingw32             
#>  ui       RTerm                       
#>  language (EN)                        
#>  collate  English_United States.1252  
#>  ctype    English_United States.1252  
#>  tz       America/Los_Angeles         
#>  date     2020-01-17                  
#> 
#> - Packages -------------------------------------------------------------------
#>  package     * version date       lib source        
#>  assertthat    0.2.1   2019-03-21 [1] CRAN (R 3.5.3)
#>  backports     1.1.5   2019-10-02 [1] CRAN (R 3.5.3)
#>  broom       * 0.5.3   2019-12-14 [1] CRAN (R 3.5.3)
#>  callr         3.4.0   2019-12-09 [1] CRAN (R 3.5.3)
#>  cli           2.0.1   2020-01-08 [1] CRAN (R 3.5.3)
#>  crayon        1.3.4   2017-09-16 [1] CRAN (R 3.5.3)
#>  desc          1.2.0   2018-05-01 [1] CRAN (R 3.5.3)
#>  devtools      2.2.1   2019-09-24 [1] CRAN (R 3.5.3)
#>  digest        0.6.23  2019-11-23 [1] CRAN (R 3.5.3)
#>  dplyr         0.8.3   2019-07-04 [1] CRAN (R 3.5.3)
#>  ellipsis      0.3.0   2019-09-20 [1] CRAN (R 3.5.3)
#>  evaluate      0.14    2019-05-28 [1] CRAN (R 3.5.3)
#>  fansi         0.4.1   2020-01-08 [1] CRAN (R 3.5.3)
#>  fs            1.3.1   2019-05-06 [1] CRAN (R 3.5.3)
#>  generics      0.0.2   2018-11-29 [1] CRAN (R 3.5.3)
#>  glue          1.3.1   2019-03-12 [1] CRAN (R 3.5.3)
#>  highr         0.8     2019-03-20 [1] CRAN (R 3.5.3)
#>  htmltools     0.4.0   2019-10-04 [1] CRAN (R 3.5.3)
#>  knitr         1.27    2020-01-16 [1] CRAN (R 3.5.2)
#>  lattice       0.20-38 2018-11-04 [2] CRAN (R 3.5.2)
#>  lifecycle     0.1.0   2019-08-01 [1] CRAN (R 3.5.3)
#>  lm.beta     * 1.5-1   2014-12-28 [1] CRAN (R 3.5.2)
#>  magrittr      1.5     2014-11-22 [1] CRAN (R 3.5.3)
#>  memoise       1.1.0   2017-04-21 [1] CRAN (R 3.5.3)
#>  nlme          3.1-137 2018-04-07 [2] CRAN (R 3.5.2)
#>  pillar        1.4.3   2019-12-20 [1] CRAN (R 3.5.3)
#>  pkgbuild      1.0.6   2019-10-09 [1] CRAN (R 3.5.3)
#>  pkgconfig     2.0.3   2019-09-22 [1] CRAN (R 3.5.3)
#>  pkgload       1.0.2   2018-10-29 [1] CRAN (R 3.5.3)
#>  prettyunits   1.1.0   2020-01-09 [1] CRAN (R 3.5.3)
#>  processx      3.4.1   2019-07-18 [1] CRAN (R 3.5.3)
#>  ps            1.3.0   2018-12-21 [1] CRAN (R 3.5.3)
#>  purrr         0.3.3   2019-10-18 [1] CRAN (R 3.5.3)
#>  R6            2.4.1   2019-11-12 [1] CRAN (R 3.5.3)
#>  Rcpp          1.0.3   2019-11-08 [1] CRAN (R 3.5.3)
#>  remotes       2.1.0   2019-06-24 [1] CRAN (R 3.5.3)
#>  rlang         0.4.2   2019-11-23 [1] CRAN (R 3.5.3)
#>  rmarkdown     2.0     2019-12-12 [1] CRAN (R 3.5.3)
#>  rprojroot     1.3-2   2018-01-03 [1] CRAN (R 3.5.3)
#>  sessioninfo   1.1.1   2018-11-05 [1] CRAN (R 3.5.3)
#>  stringi       1.4.5   2020-01-11 [1] CRAN (R 3.5.3)
#>  stringr       1.4.0   2019-02-10 [1] CRAN (R 3.5.3)
#>  testthat      2.3.1   2019-12-01 [1] CRAN (R 3.5.3)
#>  tibble        2.1.3   2019-06-06 [1] CRAN (R 3.5.3)
#>  tidyr         1.0.0   2019-09-11 [1] CRAN (R 3.5.3)
#>  tidyselect    0.2.5   2018-10-11 [1] CRAN (R 3.5.3)
#>  usethis       1.5.1   2019-07-04 [1] CRAN (R 3.5.3)
#>  vctrs         0.2.1   2019-12-17 [1] CRAN (R 3.5.3)
#>  withr         2.1.2   2018-03-15 [1] CRAN (R 3.5.3)
#>  xfun          0.12    2020-01-13 [1] CRAN (R 3.5.3)
#>  yaml          2.2.0   2018-07-25 [1] CRAN (R 3.5.3)
#>  zeallot       0.1.0   2018-01-28 [1] CRAN (R 3.5.3)
#> 
#> [1] C:/Users/I517924/Documents/R/win-library/3.5
#> [2] C:/Program Files/R/R-3.5.2/library
```

</details>

**Comment by alexpghayes** [comment]
It looks like you're using the CRAN version of `broom`. Can you check if this has been fixed in the dev version?

**Comment by ldmahoney** [comment]
Sure thing! I've never done that before so let me know if I made a mistake using the dev version.  

What I ran
```r
>library(devtools)

>dev_mode(on = T)

d>install_github("tidyverse/broom")

d> library(broom)
d> library(lm.beta)

d> mod <- stats::lm(speed ~ ., data = cars)
d> std <- lm.beta(mod)
d> tidy(std)

d> rlang::last_error()
```
What I got
```r
<error/rlang_error>
Column 6 must be named.
Use .name_repair to specify repair.
Backtrace:
  1. generics::tidy(std)
  2. broom:::tidy.lm(std)
  3. broom:::tidy.summary.lm(s)
  4. broom::fix_data_frame(co, nn[1:ncol(co)])
  6. tibble:::as_tibble.data.frame(ret)
  7. tibble:::as_tibble.list(unclass(x), ..., .rows = .rows, .name_repair = .name_repair)
  8. tibble:::lst_to_tibble(x, .rows, .name_repair, col_lengths(x))
  9. tibble:::set_repaired_names(x, .name_repair)
 14. tibble:::repaired_names(names(x), .name_repair = .name_repair)
 15. tibble:::check_unique(new_name)
Run `rlang::last_trace()` to see the full context.
d> session_info()
```
<details>
<summary>Session info</summary>

```r
devtools::session_info()
#> - Session info ---------------------------------------------------------------
#>  setting  value                       
#>  version  R version 3.5.2 (2018-12-20)
#>  os       Windows 10 x64              
#>  system   x86_64, mingw32             
#>  ui       RTerm                       
#>  language (EN)                        
#>  collate  English_United States.1252  
#>  ctype    English_United States.1252  
#>  tz       America/Los_Angeles         
#>  date     2020-01-21                  
#> 
#> - Packages -------------------------------------------------------------------
#>  package     * version    date       lib source                          
#>  assertthat    0.2.1      2019-03-21 [2] CRAN (R 3.5.3)                  
#>  backports     1.1.5      2019-10-02 [2] CRAN (R 3.5.3)                  
#>  broom       * 0.5.3.9000 2020-01-21 [1] Github (tidyverse/broom@3c922d5)
#>  callr         3.4.0      2019-12-09 [2] CRAN (R 3.5.3)                  
#>  cli           2.0.1      2020-01-08 [2] CRAN (R 3.5.3)                  
#>  crayon        1.3.4      2017-09-16 [2] CRAN (R 3.5.3)                  
#>  desc          1.2.0      2018-05-01 [2] CRAN (R 3.5.3)                  
#>  devtools      2.2.1      2019-09-24 [2] CRAN (R 3.5.3)                  
#>  digest        0.6.23     2019-11-23 [2] CRAN (R 3.5.3)                  
#>  dplyr         0.8.3      2019-07-04 [2] CRAN (R 3.5.3)                  
#>  ellipsis      0.3.0      2019-09-20 [2] CRAN (R 3.5.3)                  
#>  evaluate      0.14       2019-05-28 [2] CRAN (R 3.5.3)                  
#>  fansi         0.4.1      2020-01-08 [2] CRAN (R 3.5.3)                  
#>  fs            1.3.1      2019-05-06 [2] CRAN (R 3.5.3)                  
#>  generics      0.0.2      2018-11-29 [2] CRAN (R 3.5.3)                  
#>  glue          1.3.1      2019-03-12 [2] CRAN (R 3.5.3)                  
#>  highr         0.8        2019-03-20 [2] CRAN (R 3.5.3)                  
#>  htmltools     0.4.0      2019-10-04 [2] CRAN (R 3.5.3)                  
#>  knitr         1.27       2020-01-16 [2] CRAN (R 3.5.2)                  
#>  lifecycle     0.1.0      2019-08-01 [2] CRAN (R 3.5.3)                  
#>  lm.beta     * 1.5-1      2014-12-28 [2] CRAN (R 3.5.2)                  
#>  magrittr      1.5        2014-11-22 [2] CRAN (R 3.5.3)                  
#>  memoise       1.1.0      2017-04-21 [2] CRAN (R 3.5.3)                  
#>  pillar        1.4.3      2019-12-20 [2] CRAN (R 3.5.3)                  
#>  pkgbuild      1.0.6      2019-10-09 [2] CRAN (R 3.5.3)                  
#>  pkgconfig     2.0.3      2019-09-22 [2] CRAN (R 3.5.3)                  
#>  pkgload       1.0.2      2018-10-29 [2] CRAN (R 3.5.3)                  
#>  prettyunits   1.1.0      2020-01-09 [2] CRAN (R 3.5.3)                  
#>  processx      3.4.1      2019-07-18 [2] CRAN (R 3.5.3)                  
#>  ps            1.3.0      2018-12-21 [2] CRAN (R 3.5.3)                  
#>  purrr         0.3.3      2019-10-18 [2] CRAN (R 3.5.3)                  
#>  R6            2.4.1      2019-11-12 [2] CRAN (R 3.5.3)                  
#>  Rcpp          1.0.3      2019-11-08 [2] CRAN (R 3.5.3)                  
#>  remotes       2.1.0      2019-06-24 [2] CRAN (R 3.5.3)                  
#>  rlang         0.4.2      2019-11-23 [2] CRAN (R 3.5.3)                  
#>  rmarkdown     2.0        2019-12-12 [2] CRAN (R 3.5.3)                  
#>  rprojroot     1.3-2      2018-01-03 [2] CRAN (R 3.5.3)                  
#>  sessioninfo   1.1.1      2018-11-05 [2] CRAN (R 3.5.3)                  
#>  stringi       1.4.5      2020-01-11 [2] CRAN (R 3.5.3)                  
#>  stringr       1.4.0      2019-02-10 [2] CRAN (R 3.5.3)                  
#>  testthat      2.3.1      2019-12-01 [2] CRAN (R 3.5.3)                  
#>  tibble        2.1.3      2019-06-06 [2] CRAN (R 3.5.3)                  
#>  tidyr         1.0.0      2019-09-11 [2] CRAN (R 3.5.3)                  
#>  tidyselect    0.2.5      2018-10-11 [2] CRAN (R 3.5.3)                  
#>  usethis       1.5.1      2019-07-04 [2] CRAN (R 3.5.3)                  
#>  utf8          1.1.4      2018-05-24 [2] CRAN (R 3.5.3)                  
#>  vctrs         0.2.1      2019-12-17 [2] CRAN (R 3.5.3)                  
#>  withr         2.1.2      2018-03-15 [2] CRAN (R 3.5.3)                  
#>  xfun          0.12       2020-01-13 [2] CRAN (R 3.5.3)                  
#>  yaml          2.2.0      2018-07-25 [2] CRAN (R 3.5.3)                  
#>  zeallot       0.1.0      2018-01-28 [2] CRAN (R 3.5.3)                  
#> 
#> [1] C:/Users/I517924/Documents/R-dev
#> [2] C:/Users/I517924/Documents/R/win-library/3.5
#> [3] C:/Program Files/R/R-3.5.2/library
```
</details>

**Comment by simonpcouch** [comment]
Hi there! This appears to have been fixed in #742. A reprex after running `devtools::install_github("tidymodels/broom")`:

``` r
# load packages
library(broom)
library(lm.beta)

# fit the model
mod <- stats::lm(speed ~ ., data = cars)
std <- lm.beta(mod)

# tidy it! :-)
tidy(std)
#> # A tibble: 2 x 6
#>   term        estimate std_estimate std.error statistic  p.value
#>   <chr>          <dbl>        <dbl>     <dbl>     <dbl>    <dbl>
#> 1 (Intercept)    8.28         0        0.874       9.47 1.44e-12
#> 2 dist           0.166        0.807    0.0175      9.46 1.49e-12
```

<sup>Created on 2020-05-25 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9001)</sup>

Closing for now. Let us know if you're still running into errors.🙂
